README.md

```diff
@@ -40,7 +40,11 @@ plugins: [
       // to include analytics.page() automatically
       // if false, see below on how to track pageviews manually
       trackPage: false,
-      
+
+      // number (defaults to 50); time to wait after a route update before it should
+      // track the page change, to implement this, make sure your `trackPage` property is set to `true`
+      trackPageDelay: 300,
```

Comment:
the other values in the readme are set to their defaults. could you update this just for consistency?
```suggestion
      trackPageDelay: 50,
```

Reply:
Sure thing !